packages/kit/src/components/TxHistoryListView/TxHistoryAddressInfo.tsx

```
@@ -0,0 +1,90 @@
+      renderContent={
+        <YStack
+          gap="$2"
+          px="$3"
+          py="$2"
+          onPress={(event: GestureResponderEvent) => {
+            event.stopPropagation();
+          }}
+        >
+          <Stack
+            testID="account-network-trigger-button"
+            role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)